server/Arcadia.Assistant.Calendar/SickLeave/SendEmailSickLeaveActor.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Arcadia.Assistant.Calendar.SickLeave.Events;
+using System;
+using System.Configuration;
+using System.Threading;
+using Akka.Actor;
+using MimeKit;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Gmail.v1;
+using Google.Apis.Services;
+using Google.Apis.Gmail.v1.Data;
+
+namespace Arcadia.Assistant.Calendar.SickLeave
+{
+    public class SendEmailSickLeaveActor : UntypedActor
+    {
+        protected override void OnReceive(object message)
+        {
+            switch (message)
+            {
+                case SickLeaveIsApproved ev:
+                    SendEmail(ev);
+                    break;
+            }
+        }
+
+        private static void SendEmail(SickLeaveIsApproved Event)
+        {
+            var clientId = ConfigurationManager.AppSettings["ClientId"];
+            var clientSecret = ConfigurationManager.AppSettings["ClientSecret"];
+            try
+            {
+                var credential = GoogleWebAuthorizationBroker.AuthorizeAsync(
```

Comment:
if method returns Task, it should be `await`ed. But it probably doesn't matter since there should be not authorizationbroker in the first place :)

Reply:
Nevertheless, you should read about async/await - https://docs.microsoft.com/en-us/dotnet/csharp/async
Also, read the documentation here - https://github.com/petabridge/akkadotnet-code-samples/tree/master/PipeTo and this article https://petabridge.com/blog/akkadotnet-async-actors-using-pipeto/ - those two show you have to deal with `Task` in Akka